source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -18,16 +18,51 @@ Do _NOT_ let certificates expire. If they expire, the environment becomes non-re
 . Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
 . Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
+[IMPORTANT]
+====
+You must also renew {engine-name} certificates.
+====
 
-. Run `engine-setup` from an SSH or console session on the {engine-appliance-name} system. The `engine-setup` script prompts you with some configuration questions.
+.Renewing certificates in a standalone {engine-name}
```

Comment:
I think you should combine the manager installations into a single procedure:

1. If you are renewing certificates for a self-hosted engine, place the host in global maintenance mode....
2. Renew the certificates for the Manager...
3. Take the host out of maintenance mode...